.chloggen/extract-resources-ns.yaml

```diff
@@ -0,0 +1,19 @@
+change_type: enhancement
+
+component: instrumentation
+
+note: "Modified the logic for iterating over pod annotations, utilizing pod.GetAnnotations() to identify and extract resources using the specified namespace prefix (constants.OtelAnnotationNamespace)."
```

Comment:
I would say "introduced ability to set Otel resource attributes based on annotations for instrumentation"